pulp_ansible/tests/functional/api/collection/test_sync.py

```diff
@@ -374,3 +375,5 @@ def test_sync_progress_report(
     for pr in progress_reports:
         if pr.message == "Parsing CollectionVersion Metadata":
             assert pr.total == pr.done
+        if pr.message == "Parsing Namespace Metadata":
+            assert pr.total == pr.done
```

Comment:
```suggestion
            assert pr.total == pr.done == 1
```
I feel that we should know the total amount for this test sync since it is only one collection.